src/Resolve/maxsum.jl

```diff
@@ -235,6 +244,9 @@ function iterate!(graph::Graph, msgs::Messages, perm::NodePerm)
     for p0 in perm
         graph.ignored[p0] && continue
         maxdiff0 = update!(p0, graph, msgs)
+        if maxdiff0 isa Unsat
+            return Unsat
```

Comment:
Shouldn't this return `maxdiff0`, so as to include the same `p0` field inside the `Unsat` object?

Reply:
eeh, yes. Seems we are missing some test coverage here..